include/ur_api.h

```diff
@@ -226,6 +226,8 @@ typedef enum ur_function_t {
     UR_FUNCTION_BINDLESS_IMAGES_IMPORT_EXTERNAL_MEMORY_EXP = 226,         ///< Enumerator for ::urBindlessImagesImportExternalMemoryExp
     UR_FUNCTION_BINDLESS_IMAGES_IMPORT_EXTERNAL_SEMAPHORE_EXP = 227,      ///< Enumerator for ::urBindlessImagesImportExternalSemaphoreExp
     UR_FUNCTION_ENQUEUE_NATIVE_COMMAND_EXP = 228,                         ///< Enumerator for ::urEnqueueNativeCommandExp
+    UR_FUNCTION_SET_LOGGER_CALLBACK = 230,                                ///< Enumerator for ::urSetLoggerCallback
```

Comment:
Is this an intentional jump from `228` to `230`?

Reply:
Nope! I must've made a mistake there when rebasing, I'll get that corrected. Thanks :)